var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -5,6 +5,8 @@
 
 from spack import *
 import sys
+import shutil
+import os
```

Comment:
not used?

Reply:
Yeah, used in deleted code. I'll pull those out.